google/cloud/bigquery/v2/minimal/internal/job_client.h

```diff
@@ -66,6 +66,11 @@ class JobClient {
   StreamRange<ListFormatJob> ListJobs(ListJobsRequest const& request,
                                       Options opts = {});
 
+  /// Starts a new asynchronous BigQuery job. For more details on BigQuery
```

Comment:
It does not matter for this PR, and it does not matter because this is not going to public docs, but Doxygen style is:

```cc
/// A one line brief description.
///
/// A longer multi-line description.
/// With more details!
```

Doxygen supports both `///`-style and `/** */`-style comments. We prefer the second for things with a long description. Maybe like so:

```cc
/**
 * Starts a new asynchronous BigQuery job.
 *
 * @see https://cloud.google.com/bigquery/docs/jobs-overview
 */
```